packages/components/carousel/src/main.vue

```diff
@@ -355,6 +355,17 @@ export default defineComponent({
       }
     )
 
+    const stopWatchItems = watch(
+      () => items.value,
+      (val) => {
+        if (val.length) {
+          playSlides()
+          stopWatchItems()
+        }
+      },
+      { immediate: true, deep: true }
+    )
```

Comment:
**Question:** can we put them into `onMounted`? Can we write some tests for this change?

Reply:
Yes, sure